ssl_verify_client.t

```diff
@@ -53,7 +53,7 @@ http {
         ssl_certificate 1.example.com.crt;
 
         ssl_verify_client on;
-        ssl_client_certificate 2.example.com.crt;
+        ssl_client_certificate 2.example.com.bundle;
```

Comment:
This touches unrelated parts, which will result in poor diagnostics.
I suggest writing a separate server block that specifies ssl_client_certificate with duplicate names.

Reply:
I agree that a separate server block is better.